tutorials/forest-cover-loss-estimation/index.md

```diff
@@ -0,0 +1,167 @@
+author: nkeikon
+tags: forest, deforestation, mmu, ghg, redd, mrv, frel, frl
+date_published: 2019-08-31
+---
+
+When estimating forest cover, deforestation, or emissions from land-use change at the national level, countries use forest definitions. In this tutorial, you will learn how to extract forest area by applying technical thresholds of canopy cover and minimum area requirements. 
+
+## Context
+
+FAO defines forests as "Land spanning more than 0.5 hectares with trees higher than 5 meters and a canopy cover of more than 10 percent, or trees able to reach these thresholds in situ. It does not include land that is predominantly under agricultural or urban land use" (FAO 2015). 
+
+In various reporting of forest and/or land-use related data, countries may have different forest definitions, including these parameters, forest types or land use categories. 
+
+## Instructions
+### Select a country and set parameters
+
+This tutorial selected Bolivia as an example, with the minimum canopy cover of 10% and minimum forest area of 0.5 ha (the national definition may be different). 
+```js
+var country = 'Bolivia'; // selected country (e.g. Bolivia)
+var cc = ee.Number(10); // canopy cover percentage (e.g. 10%)
+var pixels = ee.Number(6); // minimum forest area in pixels (e.g. 6 pixels, approximately 0.5 ha in this example)
+var lossPixels = ee.Number(6); // minimum mapping area for tree loss (usually same as the minimum forest area)
+```
+
+Depending on the location of the country, the number of pixels that make up for 0.5 ha would differ. You can adjust this by calculating the actual minimum areas you were using (see step 6 below).
+
+### Tree cover
+
+Currently, Google Earth Engine has several tree cover datasets in the catalogue, including the Global Forest Change (GFC) (year 2000) and GLCF: Landsat Tree Cover Continuous Fields (2000, 2005, and 2010). Here, we use the Global Forest Change dataset. 
+```js
+var gfc2018 = ee.Image('UMD/hansen/global_forest_change_2018_v1_6');
+```
+
+1. Select 'treecover2000' in the Global Forest Change dataset. Use selfMask() to mask value zero 
+```js
+var canopyCover = gfc2018.select(['treecover2000']).selfMask();
+```
+2. Apply the minimum canopy cover percentage (e.g. greater than or equal to 10%)
+```js
+var canopyCover10 = canopyCover.gte(cc);
+```
+3. Apply the minimum area requirement using connectedPixelCount (e.g. greater than or equal to 6 pixels)
+```js
+// Use connectedPixelCount() to get contiguous area.
+var contArea = canopyCover10.connectedPixelCount();
+// Apply the minimum area requirement. 
+var minArea = contArea.gte(pixels).selfMask();
+```
+4. Scale the results in nominal value based on to the dataset's projection to display on the map
+```js
+var prj = gfc2018.projection();
+var scale = prj.nominalScale();
+Map.addLayer(minArea.reproject(prj.atScale(scale)), {
+    palette: ['#96ED89']
+}, 'tree cover: >= min canopy cover & area (light green)');
+```
+5. Calculate the tree cover area (ha)
```

Comment:
I actually define 'country' in the beginning - to avoid confusion I moved this part up.

Reply:
@nkeikon

Sorry, I should have been more clear here. This comment is regarding the markdown file. These two lines are missing from the .md file code blocks. If someone is copying and pasting code blocks from the .md file into the Editor, they will get an error when running the reduce region sections because "selected" does not exist (in the .md file). 

Having a .js file and an .md file is a troublesome and it takes significant effort to keep the two in sync while editing. The reason for having the two is that the .md format is easier to describe code blocks and add extra context (images, etc), while the .js file makes it easier for someone to "read" the script, use variable highlighting, find, and alter for their own use.

Do you have any thoughts on this - having an .md and .js file? How was your experience writing this tutorial? Is there anything that would make it easier or that you would want someone writing their first tutorial to know about?